orc8r/cloud/go/services/configurator/types.go

```diff
@@ -439,6 +439,14 @@ type EntityLoadCriteria struct {
 
 	LoadAssocsToThis   bool
 	LoadAssocsFromThis bool
+
+	// The following parameters allow for pagination of entity loads. These
+	// load criteria parameters should be used in combination with one another.
+
+	// Maximum size of entities returned
```

Comment:
Here and elsewhere -- follow go doc comment conventions, e.g. "PageSize is the maximum number of entities returned per load request."